src/components/Content/Content.css

```diff
@@ -0,0 +1,178 @@
+
+body {
+  background-color: #fff;
+}
+main {
+  display: grid;
+  grid-template-columns: 1fr repeat(12, minmax(auto, 3.75rem)) 1fr;
+  grid-gap: 2.5rem;
+  padding: 1.875rem 0 ;
+}
+.text--medium {
+  font-family: sans-serif;
+  font-size: 0.75rem;
+  line-height: 1.25rem;
+  font-weight: 400;
+  align-self: flex-end;
+  padding: 0.125rem;
+}
+.cards {
+  grid-column: 2 / span 12;
+  display: grid;
+  grid-template-columns: repeat(12, minmax(auto, 3.75rem));
+  grid-gap: 2.5rem;
+}
+.card {
+  grid-column-end: span 4;
+  display: flex;
+  flex-direction: column;
+  cursor: pointer;
+  transition: all 0.3s ease 0s;
+  box-shadow: 0 0.25rem 0.5rem 0 rgba(0, 0, 0, 0.2), 0 0.375rem 1.25rem 0 rgba(0, 0, 0, 0.19);
+  text-align: center;
+}
+.card:hover {
+  transform: translateY(-0.4375rem);
+}
+.card_image-container {
+  width: 100%;
+  padding-top: 56.25%;
+  overflow: hidden;
+  position: relative;
+}
+.card_image-container img {
+  width: 100%;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+.card_image-container:hover .overlay {
+  height: 100%;
+}
+.overlay {
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background-color: #fff;
+  overflow: hidden;
+  width: 100%;
+  height: 0;
+  transition: .5s ease;
+}
+.save-button {
+  padding:0.5em 3em;
+  background-color: #fff;
+  border:0.16em solid rgb(173, 172, 172);
+  margin:0;
+  box-sizing: border-box;
+  text-decoration:none;
+  text-transform:uppercase;
+  font-family:'Roboto',sans-serif;
+  font-weight:400;
+  color:rgb(134, 133, 133);
+  text-align:center;
+  transition: all 0.15s;
+  max-height: 2.5rem;
+  margin-top: 3rem;
+}
```

Comment:
This block still has some `em`'s in it. Convert these to `rem`.

Reply:
Done- reviewed all css files and made changes so all units are rem.